state-sync/tests/test_harness.rs

```diff
@@ -56,6 +56,7 @@ use network::{
 };
 use network_builder::builder::NetworkBuilder;
 use once_cell::sync::Lazy;
+use proptest::std_facade::HashSet;
```

Comment:
think your IDE messed up, should be `std::collections::HashSet` : p

Reply:
Yeah, IntelliJ keeps doing weird stuff like this, I really don't like it.

I've fixed it locally, just not on the push